src/runtime/3.5/windowsservercore-20H2/Dockerfile

```diff
@@ -2,20 +2,20 @@
 
 FROM mcr.microsoft.com/windows/servercore:20H2-amd64
 
-# Enable detection of running in a container
-ENV DOTNET_RUNNING_IN_CONTAINER=true `
-    # Ngen workaround: https://github.com/microsoft/dotnet-framework-docker/issues/231
+ENV `
+    # Enable detection of running in a container
+    DOTNET_RUNNING_IN_CONTAINER=true `
     COMPLUS_NGenProtectedProcess_FeatureEnabled=0
 
 RUN `
     # Install .NET Fx 3.5
-    curl -fSLo microsoft-windows-netfx3.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/microsoft-windows-netfx3-2009.zip `
+    curl -fSLo microsoft-windows-netfx3.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/microsoft-windows-netfx3-20H2.zip `
```

Comment:
I uploaded this new artifact as well as the ltsc2019 named zip.  These new names are needed for the way the Dockerfiles are generated.  Aside from this need, it is also useful to align the artifact names with the public version names.